driver/src/price_finding/price_finder_interface.rs

```diff
@@ -9,6 +9,15 @@ pub struct Fee {
     pub percentage: f64,
 }
 
+impl Default for Fee {
+    fn default() -> Self {
+        Fee {
+            token: 0,
+            percentage: 0.001,
```

Comment:
This fee seems pretty low. Are we sure this is a percentage?

Reply:
You are right it's not. It's 0.1% - what's a better name here?

Reply:
multiplier or just value? Not an easy one :)

Reply:
Discussed with @twalth3r who also has this naming issue and we will change it to feeRatio (will be a different PR though)

Reply:
ratio? what makes it a ratio?